assets/scss/components/_copy-text.scss

```diff
@@ -0,0 +1,17 @@
+.copy-text {
```

Comment:
would it be possible to remove the `border-left` grey styling on the `govuk-details__text` component as well? It looks like the prototype doesn't include that, and would be a bit cleaner without it...

Reply:
would adding `border-left: none` to the `.govuk-details__text` class here override the styles from the gov uk component? not sure how the order of precedence works...

Reply:
I'm not sure but it feels a bit risky to me. Would it not be better to use the html component in this case, and then add an additional class to that div to get rid of the border?

Reply:
actually scratch that, I didn't realise you could add classes to the nunjucks macros 👀